libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/internal/navigation/RouteProcessorRunnableTest.java

```diff
@@ -33,7 +34,58 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+class DirectionsRouteTest extends DirectionsRoute{
```

Comment:
We can convert this test to Kotlin and use `Mockk` in order to avoid adding this test double.